scripts/jenkins/ardana/ansible/roles/heat_stack/defaults/main.yml

```diff
@@ -17,7 +17,7 @@
 
 heat_action: "create"
 
-heat_stack_create_timeout: 480
+heat_stack_create_timeout: 900
```

Comment:
same here

Reply:
check this job history to get an idea why: https://ci.suse.de/job/openstack-cloud-heat-susecloud/